contributing/code/standards.rst

```diff
@@ -167,6 +167,29 @@ Service Naming Conventions
 
 * A group name uses the underscore notation.
 
+.. _contributing-code-standards-comparisons:
+
+Comparisons
+-----------
+
+Use `identical comparison`_ when the expected value must match a specific type:
```

Comment:
I agree, should be something like "Always use strict comparison unless you need type juggling".